_sass/_social-box.scss

```
@@ -0,0 +1,37 @@
+.social-box {
```

please pretend you never saw this file

```suggestion
.social-box {
```